public/app/session-persist.js

```
@@ -1,5 +1,69 @@
+    }
+  }
+};
+
+// Initialize window.VIP_PARAMS as the loaded store.
+// Existing code that reads window.VIP_PARAMS[key] keeps working.
+if (typeof window !== 'undefined') {
+  if (!window.VIP_PARAMS) window.VIP_PARAMS = SessionPersist.loadAll();
+  window.SessionPersist = SessionPersist;
+}
+
+if (typeof module !== 'undefined' && module.exports) {
```

ESLint

[error] 'module' is not defined.

(no-undef)